### DISCUSSION TITLE
How do values in config.toml handle conflicts with *arr settings?

### DISCUSSION THREAD

**Discussion by kstlouis** [Feramance/qBitrr#93] [Q&A] [answered]
Apologies if this is covered elsewhere, or if I'm misunderstanding the config's descriptions. 

It seems like some of qbittr's config settings for the *arr apps are duplicative of the application's settings, and I'm curious to know which takes precedence if there's a conflict. For instance, the `[Sonarr-TV.EntrySearch]` section -- many/all of these options seem redundant to Sonarr's GUI settings.

If I have, say `DoUpgradeSearch = false` (which is the default) in this config, but have `Allow Upgrades` enabled in Sonarr, are these in conflict? If so who wins?

**Comment by kstlouis** [comment]
I suppose a followup question would be -- for any settings in `config.toml` I don't wish to use, can I just comment them out? or will qbitrr throw any errors if there are settings it expects to see but can't find?

**Comment by Feramance** [comment, marked as answer]
So the config options are irrelevant of Sonarr/Radarr. To give you an example with the upgrade search you pointed out, the Arr apps will check for upgrades on RSS sync, whereas if you enable upgrade search in qBitrr, then qBitrr will tell the Arrs to force search everything. qBitrr works independently from the Arrs, it just sends automated API requests depending on your config.

With regards to settings you don't want to use, qBitrr has defaults set for all the 'required' settings which it will use if not defined. The defaults are the values present in the config sample. And just to clarify, the 2 Sonarr and 2 Radarr instances in the config are there as examples, you can remove them all, or leave only one, or add any number of additional instances and qBitrr will operate accordingly.
